pom.xml

```diff
@@ -2104,6 +2109,13 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents..client5</groupId>
```

Comment:
There appears to be a small typo in this groupId entry, where one area has two dots instead of one. That partly explains the problems with some of the HTTP-related unit tests; I'll provide more details on further needed follow-up in the user story.